src/tests/transfer.test.ts

```diff
@@ -178,7 +317,11 @@ describe('Transfer Action', () => {
           const result = await fundedTa.transfer({
             fromChain: 'sepolia' as any,
             toAddress: receiver.address,
-            amount: '0.001', // 0.001 ETH
+            amount: parseEther('0.0001'), // 0.0001 ETH
```

Comment:
```suggestion
            amount: parseEther('0.0001'),    
```

Remove comment as it's obvious it's `0.0001` ETH